tools/swift-plugin-server/CMakeLists.txt

```diff
@@ -1,10 +1,21 @@
 if (SWIFT_BUILD_SWIFT_SYNTAX)
+  # override the remote SwiftSystem and ArgParser dependencies in WasmKit
+  FetchContent_Declare(SwiftSystem SOURCE_DIR "${PROJECT_SOURCE_DIR}/../swift-system")
+  FetchContent_Declare(ArgumentParser SOURCE_DIR "${PROJECT_SOURCE_DIR}/../swift-argument-parser")
```

Comment:
We might end up having to do something more bespoke here to get things building on all of the platforms, but let's try this to start!